tests/cmds/test_legal_hold.py

```diff
@@ -575,6 +629,28 @@ def test_list_with_csv_format_returns_no_response_when_response_is_empty(
     assert "Matter ID,Name,Description,Creator,Creation Date" not in result.output
 
 
+def test_events_shows_events_that_respect_type_filters(
+    runner, cli_state, get_all_events_success, all_events_response
+):
+
+    cli_state.sdk.legalhold.get_all_events.return_value = all_events_response
+    result = runner.invoke(
+        cli, ["legal-hold", "events", "--event-type", "HoldCreated"], obj=cli_state
+    )
+
+    assert "564564654566" in result.output
+    assert "74533457745" not in result.output
+
+
+def test_events_with_csv_returns_no_events_when_response_is_empty(
+    runner, cli_state, get_all_events_success, empty_events_response
+):
+    cli_state.sdk.legalhold.get_all_events.return_value = empty_events_response
+    result = runner.invoke(cli, ["legal-hold", "events", "-f", "csv"], obj=cli_state)
+
+    assert "Matter ID,Name,Description,Creator,Creation Date" not in result.output
+
+
```

Comment:
Could we get test(s) for `begin_date` / `end_date` validation?
It could be to just assert the values were passed into py42 correctly.

Reply:
I'll work on this. I had tried this initially, but had trouble getting a test to recognize the date inputs in the runner.

Reply:
I added a test titled `test_search_events_is_called_with_expected_begin_timestamp`